src/pytorch_lightning/callbacks/sanity_serving.py

```diff
@@ -0,0 +1,92 @@
+import time
+from modulefinder import Module
+from multiprocessing import Process
+from typing import Any, Callable, Dict, Optional, Tuple
+
+import requests
+import torch
+
+import pytorch_lightning as pl
+from pytorch_lightning.callbacks import Callback
+from pytorch_lightning.utilities.imports import requires
+from pytorch_lightning.utilities.rank_zero import rank_zero_only
+
+
+class ServableModule(Module):
+    def configure_payload(self) -> Dict[str, Any]:
+        ...
+
+    def configure_inputs_outputs(self) -> Tuple[Dict[str, Callable], Dict[str, Callable]]:
+        ...
+
+    def serve_step(self, *args, **kwargs):
+        ...
+
+
+def _start_server(servable_model: ServableModule, host: str, port: int, method: bool) -> None:
```

Comment:
Shouldn't this function be part of the Callback so it can be overridden?

Reply:
Done !